projects/packages/my-jetpack/_inc/components/product-detail-table/index.jsx

```diff
@@ -120,31 +125,39 @@ const ProductDetailTableColumn = ( {
 				/* dummy arg to avoid bad minification */ 0
 		  );
 
-	const callToAction =
-		customCallToAction ||
-		( isFree
-			? __( 'Start for Free', 'jetpack-my-jetpack' )
+	const productMoniker = name && preferProductName ? name : title;
```

Comment:
Haha, I had to lookup what the meaning/definition of Moniker is... I never heard that term before! :laughing:
Anyway... makes sense. :+1: